test/v2/config/environment/testnet.json

```diff
@@ -0,0 +1,18 @@
+{
+  "SRSPath": "~/.test-v2/srs",
+  "KeyPath": "~/.test-v2/testnet-private-key.txt",
+  "DisperserHostname": "disperser-testnet-holesky.eigenda.xyz",
+  "DisperserPort": 443,
+  "EthRPCURLs": [
+    "https://ethereum-holesky-rpc.publicnode.com"
+  ],
+  "BLSOperatorStateRetrieverAddr":          "0x5322F480c6a14aBA6491974fa0fbF2A925C71B94",
+  "EigenDAServiceManagerAddr":              "0xD4A7E1Bd8015057293f0D0A557088c286942e84b",
+  "EigenDACertVerifierAddressQuorums0_1":   "0xFe52fE1940858DCb6e12153E2104aD0fDFbE1162",
+  "EigenDACertVerifierAddressQuorums0_1_2": "0xFe52fE1940858DCb6e12153E2104aD0fDFbE1162",
+  "EigenDACertVerifierAddressQuorums2":     "0xFe52fE1940858DCb6e12153E2104aD0fDFbE1162",
```

Comment:
If you set the addresses to empty string for contracts that haven't been deployed, the corresponding tests should automatically be skipped, instead of failing